pegasus_wrapper/nuke_checkpoints.py

```diff
@@ -21,3 +21,5 @@
     for ckpt_file in root_dir.rglob("___ckpt"):
         print(f"Removing {ckpt_file}")
         ckpt_file.unlink()
+
+# Should we move this file to scripts?
```

Comment:
@gabbard Should this `nuke_checkpoints` file be relocated into the scripts for this module?

Reply:
Yes, please do